src/sentry/workflow_engine/types.py

```diff
@@ -55,6 +55,11 @@ def bulk_get_query_object(data_sources) -> dict[int, T | None]:
 
 
 class DataConditionHandler(Generic[T]):
+    @staticmethod
+    def comparison_json_schema() -> dict[str, Any]:
```

Comment:
not sure how to have an abstract static property 🤔

Reply:
🤔 does it need to be static? could we do something like this: https://github.com/getsentry/sentry/pull/83276/files#diff-3f0b18159f9863fd3bbfdce5d8d8374a8a7473fc4808031dd520f4a83e75dd66R65-R67 ?

Reply:
i think i assumed we don't want to initialize it, but we can! added abstract property